src/test/java/uk/gov/ons/ctp/common/domain/UniquePropertyReferenceNumberTest.java

```diff
@@ -8,7 +8,7 @@
 import org.junit.Test;
 
 public class UniquePropertyReferenceNumberTest {
-  private static final String UPRN_STR = "334111111111";
+  private static final String UPRN_STR = "3341111111111";
```

Comment:
While you are here I think it's worthwhile adding tests for the exception paths for code coverage

Reply:
I have availed myself of the wonderful opportunity offered, not previously grasped by those before me, and added some delightful unit tests 😄